cmd/server/assets/apikeys.html

```diff
@@ -52,30 +57,36 @@ <h2>API keys</h2>
             <th scope="col">Key</th>
             <th scope="col">Type</th>
             <th scope="col">Enabled</th>
+            <th scope="col">Issued (1d)</th>
+            <th scope="col">Issued (7d)</th>
+            <th scope="col">Issued (30d)</th>
           </tr>
         </thead>
         <tbody>
-        {{range .apps}}
+          {{range .apps}}
           <tr>
-            <td>{{.Name}}</td>
-            <td width="100">
-              <div class="text-monospace text-truncate" style="max-width: 75px;">
-                {{.APIKey}}
-              </div>
+            <td scope="row">{{.Name}}</td>
+            <td><code>{{.APIKey}}</code></td>
+            <td>{{if .IsAdminType}}<span class="badge badge-pill badge-primary" data-toggle="tooltip"
+                data-placement="top" title="Can be used to issue verification codes">Admin</span>{{end}}
+              {{if .IsDeviceType}}<span class="badge badge-pill badge-secondary" data-toggle="tooltip"
+                data-placement="top"
+                title="For use in mobile apps to verify codes and get certificates">Device</span>{{end}}
+            </td>
+            <td>{{if .DeletedAt}}<span class="badge badge-pill badge-danger">revoked</span>
+              {{else}}<span class="badge badge-pill badge-success">enabled</span>{{end}}
             </td>
-            <td width="50">
-              {{if .IsAdminType}}<span class="badge badge-pill badge-primary" data-toggle="tooltip" data-placement="top" title="Can be used to issue verification codes">Admin</span>{{end}}
-              {{if .IsDeviceType}}<span class="badge badge-pill badge-secondary" data-toggle="tooltip" data-placement="top" title="For use in mobile apps to verify codes and get certificates">Device</span>{{end}}
+            <td scope="row">
+              {{index $codesGenerated1d .ID}}
             </td>
-            <td width="50">
-              {{if .DeletedAt}}
```

Comment:
Also looks like it was lost in a rebase

Reply:
Should be resolved now. Most of the difference was my editor changing line breaks. But there were some erroneous 'width' and 'type=row' bits that seem to be off from a few merges.